src/ServerDriver_OSVR.cpp

```diff
@@ -86,9 +87,13 @@ vr::EVRInitError ServerDriver_OSVR::Init(vr::IVRDriverContext* driver_context)
 
     trackedDevices_.emplace_back(std::make_unique<OSVRTrackedHMD>(*(context_.get())));
     trackedDevices_.emplace_back(std::make_unique<OSVRTrackingReference>(*(context_.get())));
+    trackedDevices_.emplace_back(std::make_unique<OSVRTrackedController>(*(context_.get()), 0)); // left hand
```

Comment:
`    OSVRTrackedDevice(context, vr::TrackedDeviceClass_Controller, "OSVR controller"+std::to_string(controller_index)), controllerIndex_(controller_index)
`
That is the constructor that uses the index.  For OSVRTrackedDevice, its passed a string so the name probably doesn't matter.

There is a conditional statement that uses the value of the controllerIndex_ but certainly, that would be easily to change to RightHand.  I will make the change now and check if that works.